### DISCUSSION TITLE
insert size is very small compared to read length

### DISCUSSION THREAD

**Discussion by fanlu-f** [ablab/spades#833] [Q&A] [answered]
### Description of bug

My Command is: ./metaspades.py	--pe1-1	/home/user/Desktop/data/HLP/dataRfoodW/READ_QC/R384/final_pure_reads384_1.fastq	--pe1-2	/home/user/Desktop/data/HLP/dataRfoodW/READ_QC/R384/final_pure_reads384_2.fastq	-o	/data2/FL/ASSEMBLY/ASSEMBLYWR384	-t	40

The scaffolds.fasta file is in the output file. But that said SPAdes pipeline finished WITH WARNINGS! So what should I do to solve this warning.

### spades.log

[spades.log](https://github.com/ablab/spades/files/7064414/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/7064419/params.txt)

### SPAdes version

SPAdes 3.15.2

### Operating System

LInux-5.8.0-53-generic-x86_64-with-debian-bullseye-sid

### Python Version

Python 3.6.13

### Method of SPAdes installation

manual

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment, marked as answer]
Well, as stated in the warning – the insert size is quite small and your paired end reads do almost overlap. The only way to "fix" this warning is to re-sequence your data with wider insert / fragment length used during the library preparation. You may want to forward this question to your sequencing / wet lab facility.
